egs/madcat_ar/v1/local/create_line_image_from_page_image_word_stitching.py

```diff
@@ -0,0 +1,243 @@
+#!/usr/bin/env python3
```

Comment:
These scripts are not very small. I guess it's better to follow the standard style as in `image/get_allowed_lengths.py` and maybe even do logging.

Reply:
This was an old script, I removed it. I will try to follow the style of image/get_allowed_length.py for create_line_image_from_page_image.py